app/controllers/portal_controller.rb

```diff
@@ -85,6 +96,15 @@ def media
     end
   end
 
+  # GET /record/:id/annotations
+  def annotations
+    _response, document = fetch(doc_id)
```

Comment:
This results in a superfluous call to the Record API and should not be necessary solely to retrieve annotations.